elections/election-2022.md

```diff
@@ -0,0 +1,102 @@
+# Project nomination for BSC representatves 2022
+
+In accordance with the [Elections Schedule](./schedule.md) and the
+[eBPF Foundation Charter][charter], this page holds a public record of the
+steps undertaken to nominate members for the BSC in 2022.
+
+The following members' terms will expire in 2022. The Section 4.b column
+indicates which projects must be considered to draw nominations from.
+
+| Name    | Section 4.b  |
+| ------- | ------------ |
+| (empty) | eBPF runtime |
+| TBD     |              |
+
+## Nominations (ETA August)
+
+The following nominees are under consideration for election to the BSC.
+
+### Kernel Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.i Two Linux Kernel representatives
+from the group of eBPF Linux kernel maintainers (as specified in the Linux eBPF
+maintainer file). The Linux Kernel representatives must not be employed by the
+same Company or by Related Companies._
+
+* TBD
```

Comment:
Comment: ideally representatives in each category would be split, where appropriate, between election years so that that is some continuity within each category.  So ideally 1 of the 2 in this category would be up for reelection each year, and same for other categories below.